examples/platforms/nrf52840/README.md

```diff
@@ -93,6 +90,16 @@ single transfer size limitation.
 
 [spi-hdlc-adapter]: https://github.com/openthread/openthread/tree/master/tools/spi-hdlc-adapter
 
+### Optional prefix for compiler command
```

Comment:
I got a bit confused with all the changes to section hierarchy. Is having this section as a level-3 subsection of Native SPI Slave support correct?

Reply:
fixed together with the USB dongle section